### PR TITLE
Change hardcode dcp2 prod release import script

### PR DESCRIPTION
**Pull request by eparkmonster**
## Why
Easier to determine which imports are part of what release
[Relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1677)

## This PR
Changing dcp2 prefix not hardcoded

### CODE REVIEW

**Review by quazi-broad — APPROVED** [review]
changes look good to me!

File: scripts/release_tools/import_prod.sh [hunk 1]
```diff
@@ -9,23 +9,25 @@
...
-STAGING_BUCKET=dcp2_$(gdate +%Y_%m_%d_%H%M%S)
+STAGING_BUCKET_PREFIX=$3
+STAGING_BUCKET=${STAGING_BUCKET_PREFIX}_$(gdate +%Y_%m_%d_%H%M%S)
 
 # always run these imports in our prod k8s cluster
 CURRENT_CLUSTER=$(kubectl config current-context)
-if [ $CURRENT_CLUSTER != "gke_mystical-slate-284720_us-central1-c_hca-cluster" ] ; then
-  echo "Connecting to HCA prod cluster..."
+if [[ $CURRENT_CLUSTER != "gke_mystical-slate-284720_us-central1-c_hca-cluster" ]] ; then
+  echo "Connecting to HCA  prod cluster..."
```
Comment (aherbst-broad) [inline scripts/release_tools/import_prod.sh:26]: What is the change here? If it's just an errant white space please back it out

File: scripts/release_tools/import_prod.sh [hunk 2]
```diff
@@ -9,23 +9,25 @@
...
   gcloud container clusters get-credentials hca-cluster --zone us-central1-c --project mystical-slate-284720
 fi
```
Comment (aherbst-broad) [inline scripts/release_tools/import_prod.sh:28]: I like the idea of echoing this for diagnostic purposes, can you add a bit more context to this message?

**Review by aherbst-broad — APPROVED** [review]
lgtm

**Pull request merged**